oas_docs/output/kibana.serverless.yaml

```diff
@@ -70624,211 +70624,6 @@ paths:
       x-metaTags:
         - content: Kibana, Elastic Cloud Serverless
           name: product_name
-  /api/streams/{name}/dashboards:
```

Comment:
These APIs will also be removed from our public documentation. I understand they are experimental, but it is possible some serverless users are using them and they will be gone with the next release. Just wanted to call that out!